decidim-dev/lib/decidim/dev/test/rspec_support/webmock.rb

```diff
@@ -0,0 +1,2 @@
+require "webmock/rspec"
+WebMock.allow_net_connect!
```

Comment:
We had this discussion before... and I don't remember the conclusion. Help @josepjaume XD